clients/apple/Shared/Stateful Logic/FileService.swift

```diff
@@ -180,3 +186,9 @@ class FileService: ObservableObject {
         }
     }
 }
+
+enum FileAction {
+    case move
```

Comment:
maybe you want to add create here? 

Not needed for documents, but maybe for folders?